ivadomed/testing.py

```diff
@@ -264,7 +264,8 @@ def run_inference(test_loader, model, model_params, testing_params, ofolder, cud
                         imed_inference.pred_to_png(output_list,
                                                    target_list,
                                                    fname_pred.split("_pred.nii.gz")[0],
-                                                   suffix="_pred.png")
+                                                   suffix="_pred.png",
+                                                   max_value=len(target_list))
```

Comment:
I think we have a misunderstanding in one of our previous discussion.
The `max_value` here is 1, no matter the number of classes.

`pred_to_png` is used at 3 different places in the ivadomed code:

- Here + in [main.py](https://github.com/ivadomed/ivadomed/blob/dee79e6bd90b30095267a798da99c4033cc5991b/ivadomed/main.py#L327) where the values of the `pred list` are predictions in float between [0-1], for each class, so the conversion is not related to the number of classes.
- In [evaluation.py](https://github.com/ivadomed/ivadomed/blob/dee79e6bd90b30095267a798da99c4033cc5991b/ivadomed/evaluation.py#L103) where the `pred_list` is a "painted" list (painted according to TP FP FN objects) in float with range [0-3].

So with a default `max_value` of 1, we don't have to specify the `max_value` here and we get the same behavior as before.
However, we need to specify the `max_value=3` in `evaluation.py` to have the correct output of the "painted" predictions.